wfe/wfe.go

```diff
@@ -500,11 +503,15 @@ func (wfe *WebFrontEndImpl) verifyPOST(ctx context.Context, logEvent *web.Reques
 		return nil, nil, reg, probs.Malformed("No body on POST")
 	}
 
-	bodyBytes, err := ioutil.ReadAll(request.Body)
+	bodyBytes, err := ioutil.ReadAll(&io.LimitedReader{R: request.Body, N: maxRequestSize + 1})
```

Comment:
Looks like you might actually want to use https://pkg.go.dev/net/http?tab=doc#MaxBytesReader here.